Gemfile.lock

```diff
@@ -166,6 +167,10 @@ DEPENDENCIES
   rails (~> 4.2.5)
   sass-rails (~> 4.0.0)
   sdoc
+  shoulda-context
   simple_form
   turbolinks
   uglifier (>= 1.3.0)
+
+BUNDLED WITH
+   1.10.6
```

Comment:
this is probably where you're getting the merge conflict, make sure you `rebase` / merge master to fix this.